apps/src/code-studio/components/libraries/LibraryCreationDialog.jsx

```diff
@@ -62,17 +77,25 @@ class LibraryCreationDialog extends React.Component {
           dashboard.project.getCurrentName()
         ),
         librarySource: response.source,
-        loadingFinished: true,
+        loadingState: LoadingState.DONE_LOADING,
         sourceFunctionList: libraryParser.getFunctions(response.source)
       });
     });
   };
 
   handleClose = () => {
-    this.setState({loadingFinished: false});
+    this.setState({loadingState: LoadingState.LOADING});
     this.props.onClose();
   };
 
+  copyChannelId = () => {
+    let channelId = document.getElementById('library-sharing');
+    channelId.select();
+    channelId.setSelectionRange(0, 999);
```

Comment:
Can you add a comment explaining these bounds?

Reply:
I realized that since we are supplying the channel-id, we don't have to worry about unexpected large values, so I just deleted the line.